lib/receipt.rb

```diff
@@ -0,0 +1,15 @@
+require 'order'
+
+module Receipt
+
+  def generate_receipt(order)
+    order.items.each { |item| "#{item}\n" }
+  end
+
+  def receipt_total(order)
+    total = 0
+    order.each { |item| total += item[:quantity]*item[:price].to_f}
+    "Total = #{sprintf "%.2f", total}"
```

Comment:
What does the %.2f mean?

Reply:
it makes sure that the number has 2 decimal places